src/image.rs

```diff
@@ -65,11 +66,51 @@ pub enum ImageFormat {
 }
 
 impl ImageFormat {
+    /// Return the image format specified by a path's file extension.
```

Comment:
Could we distinguish how this is function is different from the one below? Right now they have identical doc comments...

Reply:
I added examples, which are IMO the easiest to understand the difference.